extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -656,8 +654,12 @@ export class AmpStoryDevToolsTabPreview extends AMP.BaseElement {
    * @private
    * */
   repositionDevices_() {
-    const {width: layoutWidth, height} = this.getLayoutSize();
-    const width = layoutWidth * 0.8; // To account for 10% horizontal padding.
+    const {
+      offsetWidth: width,
+      offsetHeight: height,
+    } = this.element.querySelector(
+      '.i-amphtml-story-dev-tools-devices-container'
+    );
```

Comment:
Out of curiosity, what does this change do?

Reply:
Before we were measuring the part of the screen where the tab was (aka the whole screen besides the nav bar), and now we measure the device-container which occupies 80% in width (and in this PR we also removed some of the height to center the devices better), that's why we don't need the `* 0.8`. In terms of JS this just creates the two variables `width` and `height` by removing the properties `layoutWidth` and `layoutHeight` from the div.